recipes/chemper/build.sh

```diff
@@ -0,0 +1,19 @@
+cat << EOF > ${SRC_DIR}/chemper/_version.py
+import json
+
+
+version_json = '''
+{
+ "date": "2019-12-12T00:00:00-0000",
+ "dirty": false,
+ "error": null,
+ "full-revisionid": "dab37943b0a0ee063072b07f136839eae0170484",
+ "version": "1.0.0"
+}
+'''  # END VERSION_JSON
+
+def get_versions():
+    return json.loads(version_json)
+EOF
```

Comment:
This is a temporary hack to get the add version info into the tarball, since it was configured improperly due to a missing `.gitattributes` file. This will be fixed in the next released and this file removed.

Reply:
Is the JSON string needed elsewhere? Otherwise you can also return the dict directly, no parsing needed.

Reply:
Nvm, just saw this is how versioneer generates it too!